app/controllers/application_searches_controller.rb

```diff
@@ -34,6 +34,10 @@ def search_params
     )
   end
 
+  def sorted_filter_params
+    { filter: @filter.params, sorting: @sorting.params }
+  end
+
```

Comment:
These params are used by the pagination buttons. (Note: they exclude none editable params.)